website/content/docs/job-specification/resources.mdx

```diff
@@ -167,3 +187,4 @@ resource utilization and considering the following suggestions:
 [numa]: /nomad/docs/job-specification/numa 'Nomad NUMA Job Specification'
 [`secrets/`]: /nomad/docs/reference/runtime-environment-settings#secrets
 [concepts-cpu]: /nomad/docs/architecture/cpu
+[disk_throttle]: /nomad/docs/job-specification/disk_throttle 'Nomad Disk_Throttle Job Specification'
```

Comment:
```suggestion
[disk_throttle]: /nomad/docs/job-specification/disk_throttle
```

You don't need to add a page title after the link. That pattern is from a much earlier version of the docs.